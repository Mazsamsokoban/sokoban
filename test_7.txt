loadtest 7
move w1 right
showmaze