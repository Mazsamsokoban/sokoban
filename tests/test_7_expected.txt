f1:w1 f2:w2 f3:box f4:w2 

f1:w1 f2:w2 f3:box f4:w2 

